modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -155,20 +155,30 @@ conf = {{{packageName}}}.Configuration(
 
     _default = None
 
-    def __init__(self, host="{{{basePath}}}",
+    def __init__(self, host=None,
                  api_key=None, api_key_prefix=None,
                  username=None, password=None,
                  discard_unknown_keys=False,
                  disabled_client_side_validations="",
 {{#hasHttpSignatureMethods}}
                  signing_info=None,
 {{/hasHttpSignatureMethods}}
+                 server_index=None, server_variables=None,
+                 server_operation_index=None, server_operation_variables=None,
```

Comment:
This method already has 8 inputs not counting self.
Adding 4 more variables here increases out input count to 12 which is a lot.
It is clearer if related variables are grouped together in dicts.
Can you restructure these 4 new additions into two dicts?
How about something like:
```
server_info=None
# which is a dict like: {index:1, variables={}}
operation_server_info=None:
# which is a dict like {index_by_operation_id: {'get_pet': 0}, variables_by_operation_id: {'get_pet': blah}}
```

Reply:
I am trying to keep the usage similar with other languages. I don't see this as an issue. I would argue that configuration should be a mapping itself.

Reply:
I will add the docstrings.